packages/auth/src/__tests__/buildLegacyAccessPolicy.test.js

```diff
@@ -133,89 +133,89 @@ describe('buildLegacyAccessPolicy', () => {
       const user = await upsertDummyRecord(models.user);
 
       // Create a facility nested deep within a new country
-      const canada = await findOrCreateDummyRecord(
+      const greece = await findOrCreateDummyRecord(
```

Comment:
We also insert Canada entities in [testAccessPolicyHandler.js](https://github.com/beyondessential/tupaia/blob/dev/packages/auth/src/__tests__/buildAccessPolicy/testAccessPolicyHandler.js#L18). The entity existence check there may happen concurrently with the one that is triggered here. In that case both checks will fail, and both tests will attempt to create the same entity in the DB.